### ISSUE TITLE
can't even get initial run to work

### ISSUE THREAD

**Issue by shellwe** [postlight/headless-wp-starter#228]
I was getting this error so I completely deleted docker to remove any and all containers. It looks like it installed fine. I then reinstalled docker and ran localhost:3000 and I get the following error.

Unhandled Rejection (TypeError): Cannot read property 'status' of undefined

it appeared to be on this line

if (err.data.status === 403) {

The only thing since i did since I installed docker was I ran this command on admin.

docker-compose up -d

I have not changed any code or anything.

I am looking at my status and db-headless, frontend-graphql, and wp-headless are all running.

I am on Windows 10 pro and have the latest version of docker and the headless wp starter software.

**Comment by amitrathik** [comment]
I am getting the same issue.

**Comment by magidmroueh** [comment]
it seems the wordpress container is not running, wait around 2-3 minutes then try to access the site again.

**Comment by shellwe** [comment]
@magidmroueh, thank you for the feedback. I suspected the same and let it sit for over an hour in the background while I did more work. I restarted my PC and even uninstalled and reinstalled Docker so that all containers were gone (suspecting one of the containers didn't install properly) and it still didn't work.
I needed it for a presentation on the WordPress REST API but that presentation is over and I probably won't give it again due to lack of interest. I had my friend try it and it worked completely fine for him so I don't know what my deal was.
